packages/app-desktop/gui/MasterPasswordDialog/Dialog.tsx

```
@@ -24,7 +24,7 @@ enum Mode {
 	Reset = 2,
 }
 
-export default function(props: Props) {
+export default function (props: Props) {
```

ESLint

[error] Unexpected space before function parentheses.

(space-before-function-paren)